spec/requests/portfolio_items_spec.rb

```diff
@@ -335,15 +335,6 @@
       end
     end
 
-    context "when copying with a specified name" do
-      let(:params) { { :portfolio_item_name => "NewPortfolioItem" } }
-
-      it "returns the name specified" do
```

Comment:
Discussed with @lindgrenj6. He asked to remove this because the expectation on line 361 was duplicating the behavior of this test since we removed `display_name`.